rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -135,9 +135,10 @@ def _parse_airdrops(database: 'DBHandler', airdrops_data: dict[str, Any]) -> dic
     return airdrops
 
 
-def fetch_airdrops_metadata(database: 'DBHandler') -> tuple[dict[str, Airdrop], dict[str, list[str]]]:  # noqa: E501
+def fetch_airdrops_metadata(database: 'DBHandler') -> tuple[dict[str, Airdrop], dict[str, list[str]], set[str]]:  # noqa: E501
     """Fetches airdrop metadata from the rotki/data repository. If it's not cached, parses them,
     and returns them in two parts: a dict of Airdrop instances and a dict of POAP airdrops data.
+    Also returns the list of airdrops that don't have decoders yet provided by the remote index.
```

Comment:
Provided by the remote index? What do you mean?

```suggestion
    Also returns the list of airdrops that don't have decoders yet provided by the remote index.
```